docs/platform/tutorials/image-processing/index.md

```diff
@@ -147,9 +141,11 @@ There are also some additional services in the pipeline:
 
 2. *Max vehicle window* - calculates the total vehicles over a time window of one day. This service publishes messages its output topic. 
 
-3. *Data API* - this REST API service provide two endpoints: one returns the *Max vehicle window* values for the specified camera, and the other endpoint returns camera data for the specified camera. This API is called by the UI to obtain useful data.
+3. *Data buffer* - this provides a one second data buffer. This helps reduce load on the Data API service.
+
+4. *Data API* - this REST API service provide two endpoints: one returns the *Max vehicle window* values for the specified camera, and the other endpoint returns camera data for the specified camera. This API is called by the UI to obtain useful data.
```

Comment:
it has 4 endpoints currently.

 - max vehicles - last known maximum vehicle count for each camera. 24 hour rolling window.
 - detected objects - output from the computer vision service for all cameras. excludes images
 - vehicles - the last known vehicle count for each camera
 - image - the last image from the specified camera

tbh you don't have to mention all of these. perhaps what you already have is sufficient.

Reply:
OK two endpoints were additions. I can update accordingly.